omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml

```diff
@@ -1,63 +1,112 @@
 <htmlform formName="HMIS ACP 007 Missed Appointment Tracking Form" formDescription="Record all clients who missed their appointment" formEncounterType="6d88e370-f2ba-476b-bf1b-d8eaf3b1b67e" formUuid="59d9341a-f65e-4e86-a98b-415ea54dfa81" formVersion="3.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="2" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4'" formDisplayStyle="Standard" formLabel="Missed Appointment Tracking Form">
 	<script type="text/javascript">
+		if (jQuery) {
 
+		jq(document).ready(function () {
+		//Block Multiple encounter entry on same date
+		blockEncounterOnSameDateEncounter(jq("#encounterDate").find("input[type=hidden]"),"block");
 
-        if (jQuery) {
+		disableContainer('#90041');
+		disableContainer('#165398');
+		disableContainer('#164975');
+		disableContainer('#165399');
 
-			jq(document).ready(function () {
-				//Block Multiple encounter entry on same date
-        		blockEncounterOnSameDateEncounter(jq("#encounterDate").find("input[type=hidden]"),"block");
+		var $otherRows,
+			$firstRow,
+			$thisRow,
+			$nextRow,
+			$lastVisibleRow;
+		var showAddButtonForLastVisibleRow = function($container) {
+			$lastVisibleRow = $container.find('div.follow-up:visible:last');
+			console.log($lastVisibleRow);
+			$lastVisibleRow.find('.add').show();
+		};
 
-				disableContainer('#90041');
-				disableContainer('#165398');
-				disableContainer('#164975');
-				disableContainer('#165399');
+		jq('.follow-up').change(function(){
+			<!-- $otherOption= jq(this).find('[id^="165100"]').find(":checkbox").val();
```

Comment:
@SoniaRMK please remove this commented code if it is not being used anywhere

Reply:
I am going to remove it

Reply:
@solemabrothers Done!